pom.xml

```diff
@@ -133,6 +133,11 @@
       <artifactId>util</artifactId>
       <version>${rmb.version}</version>
     </dependency>
+    <dependency>
+      <groupId>com.neovisionaries</groupId>
+      <artifactId>nv-i18n</artifactId>
+      <version>1.29</version>
+    </dependency>
```

Comment:
Have you considered using Java's standard Locale instead (Locale.getISOCountries() etc.)?

Reply:
Well this library has a ready to use methods that fits well to get country name based on the code provided. With Locale I didn't find a direct way where in the country code when feeded will return its corresponding name.

Reply:
Yeah, I don't think there's a direct way, I thought about something like this
```
  public static String getCountryNameByCodeIgnoreCase(String code) {
    if (code == null) {
      return null;
    }

    String isoCode = Arrays.stream(Locale.getISOCountries())
      .map(String::cast)
      .filter(code.toUpperCase()::equals)
      .findFirst()
      .orElse(null);

    return isoCode == null ? null : new Locale("", isoCode).getDisplayName();
  };
```
But it's totally up to you, personally I just try to avoid unnecessary dependencies

Reply:
I think I have to agree with @alexanderkurash here @gurleenkaurbp. I'd use the built-in library here. Also it looks like neovisionaries i18n library hasn't had a release since 2021: https://mvnrepository.com/artifact/com.neovisionaries/nv-i18n. Seems pretty painless to convert the two char code to the full name with `new Locale("", "CN").getDisplayName()`.

Reply:
Done